tests/reference/asr-structs_10-cb8a283.stderr

```diff
@@ -0,0 +1,19 @@
+warning: Struct member functions are yet to be supported
+ --> tests/errors/structs_10.py:7:5 - 8:24
+  |
+7 |        def print_len(self: StringIO):
+  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
+...
+  |
+8 |            print(self._len)
+  | ...^^^^^^^^^^^^^^^^^^^^^^^^ Skipped defining function print_len
```

Comment:
What does it mean Struct member functions are yet to be supported? Shouldn't it rather give an error message?

Reply:
>What does it mean Struct member functions are yet to be supported? Shouldn't it rather give an error message?

I thought a `struct` is like a class, so it could/may have member functions defined inside it. Currently, we do not yet support member functions. The message warns that there is a function provided by the user inside the class and that
function is being skipped/ignored since we do not yet support defining member functions. We can as well give an error message (instead of a warning). The idea behind the warning was that the rest of the code (if semantically correct) could be compiled (and executed).

doubt: Do we plan to support member functions in `structs` in LPython? Or would `structs` only contain data members?

Reply:
Struct member functions are not supported

Reply:
And let's make it an error message, so that we get nice error messages for the current feature set. If we later decide to implement a larger subset of LPython, then we'll remove this error message and implement the feature. If we decide we don't want to do that, then we keep the error message.